lib/common/common/src/mmap_hashmap.rs

```diff
@@ -186,31 +192,20 @@ impl<K: Key + ?Sized> MmapHashMap<K> {
         self.header.buckets_count as usize
     }
 
+    pub fn keys(&self) -> impl Iterator<Item = &K> {
```

Comment:
We need to add this to the benchmark.

Reply:
I propose to keep just test coverage, I find this part not performance critical